department-of-veterans-affairs/caseflow
# Remove actionNameOfTask() helper function

Removes a utility function made redundant when we [created subclasses for each ColocatedTask action](https://github.com/department-of-veterans-affairs/caseflow/pull/11295) instead of relying on the deprecated `action` field in the `tasks` table.

### Testing Plan
1. Check out `origin/master`
1. Log in as `BVALSPORER`
1. Go to your personal queue, notice that the task types in the "Type(s)" column appear normal
1. Check out this branch
1. Return to your personal queue, notice that the task types stay the same.
